API_DOCUMENTATION.md

```diff
@@ -9,5 +9,48 @@
   **Response:**
   {body: string}
 
+  ---
+
+#### POST "/messages/"
+  *Create a new message with given JSON body information*
+  
+  **Params:**
+  {
+  text: string,
+  senderId: int,
+  receiverId: int
+  }
+
+  **Response:**
+  STATUS 200 OK
+
+#### GET "/messages/conversations/{userId}"
+  *Get ids of the users the userId identified user is having conversations with*
+  
+  **Params:**
+  userId: int
+
+  **Response:**
+  [id: int]
```

Comment:
Currently the response is ok, but will have to expand this in the future, since the UI will probably need to display the other user's name, maybe last message, time and so on.